src/checkpoint/checkpoint_txn.c

```diff
@@ -722,6 +723,15 @@ __checkpoint_stats(WT_SESSION_IMPL *session)
     /* Compute timer statistics for the checkpoint prepare. */
     msec = WT_TIMEDIFF_MS(conn->ckpt.prepare.timer_end, conn->ckpt.prepare.timer_start);
     __checkpoint_timer_stats_set(&conn->ckpt.prepare, msec);
+
+    /*
+     * Walk the list of open handles, updating the connection level count of files with available
+     * space above 50 and 90 percent respectively. Skip when using disagg, as available file space
+     * does not apply.
+     */
+    if (!__wt_conn_is_disagg(session))
```

Comment:
I thought we agreed on the previous PR to make the function return something else than void so we don't ignore a potential error from this function.